src/lexer/src_string.sig

```diff
@@ -1,7 +1,9 @@
 signature SRC_STRING =
 sig
     type yypos = int
-
+    exception StringNotClosed of int * int (* line position * line number *)
```

Comment:
The comment is just used to describe what the tuple represents. When we throw the exception we want to know where the string started(i.e., the line position and line number)